static/css/style.css

```diff
@@ -14,11 +14,6 @@ time, mark, audio, video {
 	vertical-align: baseline;
 }
 
-article, aside, details, figcaption, figure, footer, header, hgroup, menu, nav,
```

Comment:
Hmm... It seems that Opera mini may not work properly with HTML5 semantic elements without `display: block`.

[http://caniuse.com/#feat=html5semantic](http://caniuse.com/#feat=html5semantic)